manifests/satellite6.pp

```diff
@@ -0,0 +1,19 @@
+# Class: satellite6
+#   Install the SAtellite 6 flavor of the robottelo_slave
+#
+class robottelo_slave::satellite6 {
```

Comment:
Having a Puppet class name describe what the class does is the wrong approach IMO. It needs to describe what the host it is applied to does. Think of it like this - Puppet classes are "classes of hosts" not "classes of code".
"robottelo_slave" is already quite ready for building, which is why I'm hesitant about making this class name include the word 'build'.
Like I said, this class merely adds the ability for the slave to talk to Brew (As opposed to Koji which it already can). I was thinking it could be used to collect the various DS-specific configuration bits which is why named it so, but trying to take your approach, would something like "robottelo_slave::rh_internal" or "robottelo_slave::brew" work better?